tests/rendering/zelos/zelos.html

```diff
@@ -33,6 +33,10 @@
     goog.require('Blockly.blockRendering.Debug');
     goog.require('Blockly.zelos.Renderer');
     // Blockly.blockRendering.startDebugger();
+
+    Blockly.BlockSvg.prototype.setOutputShape = function() {
```

Comment:
Add a comment (here or in the other PR) that this is for testing with the PXT blocks.